deploy/catalog/templates/apiserver.yaml

```diff
@@ -48,6 +70,14 @@ spec:
   selector:
     app: apiserver
   ports:
-  - protocol: TCP
+  {{ if .Values.insecure -}}
+  - name: insecure
+    protocol: TCP
+    nodePort: {{ default 30001 .Values.insecureServicePort }}
+    port: 80
+    targetPort: {{ default 8081 .Values.insecurePort }}
+  {{- end }}
+  - name: secure
+    protocol: TCP
     port: 6443
     targetPort: 6443
```

Comment:
even tho we're focused on insecure for the moment, we might as well not totally ignore the secure path, so let's add nodePort and make the secure "targetPort" a knob too.